work/v1/types.go

```diff
@@ -66,6 +70,72 @@ type DeleteOption struct {
 	SelectivelyOrphan *SelectivelyOrphan `json:"selectivelyOrphans,omitempty"`
 }
 
+// ManifestConfigOption represents the configurations of a manifest defined in workload field.
+type ManifestConfigOption struct {
+	// ResourceIdentifier represents the group, resource, name and namespace of a resoure.
```

Comment:
Clarify in a comment that if this refers to a thing not created by this manifest work, the result will *not* be shown.

Reply:
+1